src/solver/conv_hip_implicit_gemm_bwd_data_xdlops.cpp

```diff
@@ -365,9 +365,9 @@ bool ConvHipImplicitGemmBwdXdlops::IsApplicable(const ConvolutionContext& ctx,
         return false;
     if(!problem.IsLayoutNHWC())
         return false;
-    const std::string& arch = ctx.GetStream().GetDeviceName();
-    if(!(arch == "gfx908" || arch == "gfx90a"))
+    if(!IsComposableKernelSupportedHardware(ctx))
```

Comment:
I am afraid this enables the solver not only for gfx940, but also for gfx803/64, gfx900 and gfx906.

Reply:
Oops. Fixed.

Reply:
Resolved